net/SslSocket.hpp

```diff
@@ -308,8 +308,12 @@ class SslStreamSocket final : public StreamSocket
             const unsigned long bioError = ERR_peek_error();
             if (bioError != 0)
             {
+                const size_t errorStringLength = 64;
+                char* errorString = new char[errorStringLength];
+                ERR_error_string_n(bioError, errorString, errorStringLength);
```

Comment:
Is there a reason we use 64 bytes when `ERR_error_string()` expects at least 256 bytes? I would use a 256 bytes buffer.

Reply:
I missed that ERR_error_string expected 256 bytes.